src/components/TodoList/TodoList.tsx

```diff
@@ -1 +1,19 @@
-export const TodoList = () => {};
+import React from 'react';
+import { Todo } from '../../types/types';
+import { TodoInfo } from '../TodoInfo';
+
+type Props = {
+  todos: Todo[];
+};
+
+export const TodoList: React.FC<Props> = props => {
+  const { todos } = props;
+
+  return (
+    <section className="TodoList">
+      {todos.map((todo: Todo) => {
+        return <TodoInfo key={todo.id} todo={todo} />;
+      })}
```

Comment:
```suggestion
      {todos.map((todo: Todo) => 
        <TodoInfo key={todo.id} todo={todo} />;
      )}
```
If you haven't any calculations so then you don't need `return`